
project build/
diff --git a/build/target/product/embedded.mk b/build/target/product/embedded.mk
index 33c9918..11e02ae 100644
--- a/build/target/product/embedded.mk
+++ b/build/target/product/embedded.mk
@@ -69,7 +69,6 @@ PRODUCT_PACKAGES += \
 
 # SELinux packages
 PRODUCT_PACKAGES += \
-    auditd \
     sepolicy \
     file_contexts \
     seapp_contexts \

project vendor/liquid/
diff --git a/vendor/liquid/config/common.mk b/vendor/liquid/config/common.mk
index 7a3bb48..0131afb 100644
--- a/vendor/liquid/config/common.mk
+++ b/vendor/liquid/config/common.mk
@@ -15,7 +15,6 @@ PRODUCT_PROPERTY_OVERRIDES += \
 
 # properties
 PRODUCT_PROPERTY_OVERRIDES += \
-    ro.build.selinux=1 \
     persist.sys.root_access=3
 
 # Tether for all
